HousingSearchApi.Tests/V1/E2ETests/Steps/GetAssetSteps.cs

```diff
@@ -169,6 +189,13 @@ public async Task ThenThatAddressShouldBeTheOnlyResult(string address)
             result.Results.Assets.First().AssetAddress.PostCode.Should().Be(address);
             result.Results.Assets.Count().Should().Be(1);
         }
+        public async Task ThenThatTemporaryAccomodationAddressShouldBeTheFirstResult(string address)
+        {
+            var resultBody = await _lastResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
+            var result = JsonSerializer.Deserialize<APIResponse<GetAllAssetListResponse>>(resultBody, _jsonOptions);
```

Comment:
Should the type here be `APIAllResponse` which is typically used with `GetAllAssetListResponse`?